keras_tuner/tuners/hyperband.py

```diff
@@ -383,15 +383,11 @@ def run_trial(self, trial, *fit_args, **fit_kwargs):
             fit_kwargs["initial_epoch"] = hp.values["tuner/initial_epoch"]
         return super(Hyperband, self).run_trial(trial, *fit_args, **fit_kwargs)
 
-    def _build_model(self, hp):
-        model = super(Hyperband, self)._build_model(hp)
+    def _build_hypermodel(self, hp):
+        model = super(Hyperband, self)._build_hypermodel(hp)
         if "tuner/trial_id" in hp.values:
             trial_id = hp.values["tuner/trial_id"]
-            history_trial = self.oracle.get_trial(trial_id)
             # Load best checkpoint from this trial.
-            model.load_weights(
-                self._get_checkpoint_fname(
-                    history_trial.trial_id, history_trial.best_step
-                )
-            )
+            print("Load weights from saved model", trial_id)
```

Comment:
Please remove this line. We will not print information until we have a centralized way to control the verbosity.

Reply:
Thank you for your review ! I removed the line and formatted the code as stated.